templates/core/terraform/scripts/build-deploy-ui.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+# skip if feature flag not set to true
+if [ ! "${DEPLOY_UI}" == "true" ]; then
+  echo "Skipping UI deployment as DEPLOY_UI is not true"
+  exit 0
+fi
+
+pushd ./ui/app
+
+# replace the values in the config file
+jq --arg rootClientId "${SWAGGER_UI_CLIENT_ID}" \
```

Comment:
nit: Should this be SWAGGER_UI anymore? Should we change the output of the app-reg.sh to have UX_CLIENT_ID (or similar)